test/known_issues/test-vm-data-property-writable.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/10223
+
+require('../common');
+var vm = require('vm');
```

Comment:
This will be flagged by the linter. It additionally flags that `util` on line 7 is never used.